shell/platform/android/io/flutter/plugin/common/MethodChannel.java

```diff
@@ -26,7 +26,7 @@
  * <p>The logical identity of the channel is given by its name. Identically named channels will
  * interfere with each other's communication.
  */
-public final class MethodChannel {
```

Comment:
is this to do https://www.baeldung.com/mockito-final#final-class?

Reply:
ya that's outside the scope of this PR. Left https://github.com/flutter/flutter/issues/59356. Should be an "easy" fix.